app/src/main/java/edu/uc/kansakpb/ikeahjalpare3/RetrofitClientInstance.kt

```diff
@@ -1,22 +1,22 @@
-package edu.uc.kansakpb.ikeahjalpare3.ui.main
+package edu.uc.kansakpb.ikeahjalpare3
 
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitClientInstance {
-    private var retrofit: Retrofit? = null;
-    private val BASE_URL = "https://ikeafurnituredb.s3.us-east-2.amazonaws.com"
+    private var retrofit: Retrofit? = null
```

Comment:
Remove redundant semicolon as it is not necessary in Kotlin as explained in lecture /videos.